CHANGELOG.next-release.md

```diff
@@ -13,9 +13,16 @@ This file contains all changes which are not released yet.
 <!--FIXES-END-->
 # Features and enhancements
 <!--ENHANCEMENTS-START-->
+* Remove 1000 character limit for HTTP client body capturing  - [#1234](https://github.com/elastic/apm-agent-java/pull/4058)
 
 <!--ENHANCEMENTS-END-->
 # Deprecations
 <!--DEPRECATIONS-START-->
 
 <!--DEPRECATIONS-END-->
+
+# Breaking Changes
```

Comment:
Note that this needs either https://github.com/elastic/apm-agent-java/issues/4039 to be fixed before the next release or we'll have to update the docs manually on release.

Reply:
See https://github.com/elastic/apm-agent-java/pull/4060